docs/data-management/mriqc.md

```diff
@@ -10,10 +10,96 @@
+    - [ ] If you could find the solution to the problem, re-run *MRIQC* on that particular subject implementing this solution.
+    - [ ] In case you do not understand the error message, search for associated keywords in the [issues of the *MRIQC* github repository](https://github.com/nipreps/mriqc/issues); it is likely someone else experienced the same problem before you and reported it. The solution might be documented in the issue. Don't forget to check closed issues!
+    - [ ] If that did not help, you might find help on [NeuroStars](https://neurostars.org/).
+    - [ ] If no issue has been opened regarding this error and you did not find answer in NeuroStars, then open an issue in the *MRIQC* github repository. Your description of the problem need to be as complete and detailed as possible to help the maintainers identify the problem efficiently.
+
 - [ ] Push the new derivatives to the remote storage.
     ```shell
     datalad push --to ria-storage
     datalad push --to origin
     ```
-- [ ] Screen the T1w, DWI and BOLD visual reports
-- [ ] Schedule an extra session after the initially-planned scanning period to reacquire it if either the dMRI or the RSfMRI quality is insufficient.
+
+## Visualizing *MRIQC*'s individual reports
+
+Following our protocols<sup>[1]</sup>, the quality of unprocessed images MUST be assessed before executing any processing (e.g., *fMRIPrep* or *dMRIPrep* for preprocessing).
+In addition, *MRIQC* is executed prior any further processing step considering our preliminary findings regarding *defacing*<sup>[2]</sup>.
+
+### Assessing anatomical images (T<sub>1</sub>-weighted and T<sub>2</sub>-weighted)
+- [ ] Open each *MRIQC* report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Visualize the first mosaic (background-enhanced mosaic) and apply the [exclusion criteria](qaqc-criteria.md#background-enhanced-mosaic)
+- [ ] Scroll down to the zoom in the brain mask mosaic and apply the [exclusion criteria](qaqc-criteria.md#zoomed-in-brain-mosaic)
+- [ ] Assign a quality rating and indicate artifacts with the *Rating widget*.
+    To assign a quality rating, follow the [QA/QC criteria]() stated below.
+- [ ] Download the rating file as a JSON and add it to the derivatives dataset.
+
+!!! warning "Immediately report images deemed *exclude*, as an issue in the dataset's repository"
+
+### Assessing functional images
+
+!!! danger "Insufficient quality of an fMRI run requires recalling the session"
+
+    - [ ] Immediately report images deemed *exclude*, as an issue in the dataset's repository.
+    - [ ] Proceed to scheduling an extra session after the initially-planned scanning period.
+
+#### RSfMRI
+- [ ] Open each *MRIQC* report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Visualize the first mosaic (BOLD average) and apply the [exclusion criteria](qaqc-criteria.md#bold-average)
```

Comment:
The new reports show the standard deviation first and then the carpet plot. Please check them :)